src/states/game.gd

```diff
@@ -66,11 +67,13 @@ func _ready() -> void:
 func _input(event: InputEvent) -> void:
 	if event.is_action_pressed("ui_cancel"):
 		pause_menu.open_menu()
+		my_player.release(true)
```

Comment:
Either:
```gdscript
if my_player.is_drawing_back:
    my_player.release(true)
```

Or, add a safeguard in `release` which checks if we're not drawing back and bails out early:
```
func release(is_cancel := false) -> void:
    if not is_drawing_back:
        return
```

Reply:
Por que no los dos?